.github/workflows/eden.yml

```diff
@@ -24,15 +23,20 @@ jobs:
         run: |
           sudo add-apt-repository ppa:canonical-server/server-backports
           sudo apt install -y qemu-utils qemu-system-x86 jq
-      - name: prepare eden
-        run: |
-          docker run -v $PWD:/out lfedge/eden:30c8b3f cp -a /eden/. /out/
-          sudo chown -R $(whoami) .
-          ./eden config add default
       - name: get eve
         uses: actions/checkout@v2
         with:
           path: 'eve'
+      - name: prepare eden
+        run: |
+          if [ -f ${{ github.workspace }}/eve/tests/eden/eden-version ]; then
+            EDEN_VERSION=$(cat ${{ github.workspace }}/eve/tests/eden/eden-version)
+          else
+            EDEN_VERSION=lfedge/eden:30c8b3f
```

Comment:
Will this be a hash we will have to periodically update? Or is it an error to not have an eden-version file?

Reply:
We will update it, but we still need to support old branches without this file.